Description: store SMS in /var/lib/fsogsmd/sms
 Use /var/lib/fsogsmd/sms to store SMS messages. Upstream's
 default location may be flushed during the boot. The new
 location complies with the FHS.
Author: Rico Rommel <rico@bierrommel.de>
Index: fso-gsmd/conf/default/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/default/fsogsmd.conf	2012-05-25 18:19:48.518093001 +0200
+++ fso-gsmd/conf/default/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -28,7 +28,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2"
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_singleline]
 # No settings yet
Index: fso-gsmd/conf/htc_qualcomm_dream/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/htc_qualcomm_dream/fsogsmd.conf	2012-05-25 18:19:48.550093001 +0200
+++ fso-gsmd/conf/htc_qualcomm_dream/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -27,7 +27,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2";+GTKC=2
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_qualcomm_htc]
 # No settings yet
Index: fso-gsmd/conf/htc_qualcomm_msm/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/htc_qualcomm_msm/fsogsmd.conf	2012-05-25 18:19:48.478093001 +0200
+++ fso-gsmd/conf/htc_qualcomm_msm/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -22,7 +22,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2"
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_qualcomm_htc]
 # No settings yet
Index: fso-gsmd/conf/htcleo/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/htcleo/fsogsmd.conf	2012-05-25 18:19:48.494093001 +0200
+++ fso-gsmd/conf/htcleo/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -27,7 +27,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2";@BRIC=0;+GTKC=2;+CLIP=1
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_singleline]
 # No settings yet
Index: fso-gsmd/conf/motorola_ezx/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/motorola_ezx/fsogsmd.conf	2012-05-25 18:19:48.526093001 +0200
+++ fso-gsmd/conf/motorola_ezx/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -27,7 +27,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2"
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_freescale_neptune]
 muxnode_prefix = /dev/mux
Index: fso-gsmd/conf/openmoko_gta/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/openmoko_gta/fsogsmd.conf	2012-05-25 18:19:48.458093001 +0200
+++ fso-gsmd/conf/openmoko_gta/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -62,7 +62,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2"
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.lowlevel_openmoko]
 power_node = /sys/bus/platform/devices/gta02-pm-gsm.0/power_on
Index: fso-gsmd/conf/palm_pre/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/palm_pre/fsogsmd.conf	2012-05-25 18:19:48.538093001 +0200
+++ fso-gsmd/conf/palm_pre/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -26,7 +26,7 @@
 pdp_type = ippp
 lowlevel_type = palmpre
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.lowlevel_palmpre]
 
Index: fso-gsmd/conf/GTA04/fsogsmd.conf
===================================================================
--- fso-gsmd.orig/conf/GTA04/fsogsmd.conf	2012-05-25 18:19:48.510093001 +0200
+++ fso-gsmd/conf/GTA04/fsogsmd.conf	2012-05-25 18:26:27.830093001 +0200
@@ -29,7 +29,7 @@
 modem_init = E0V1;+CMEE=1;+CRC=1;+CSCS="UCS2"
 
 # Where to store SMS: <path> (default=/tmp/fsogsmd/sms/)
-sms_storage_dir = /tmp/fsogsmd/sms/
+sms_storage_dir = /var/lib/fsogsmd/sms/
 
 [fsogsm.modem_option_gtm601]
 modem_urc_access = serial:/dev/ttyHS_Modem:115200
